app/screens/permalink/permalink.js

```diff
@@ -303,18 +333,21 @@ export default class Permalink extends PureComponent {
         }
     };
 
-    archivedIcon = (style) => {
-        let ico = null;
+    archivedIcon = () => {
+        const style = getStyleSheet(this.props.theme);
```

Comment:
`getStyleSheet` is called inside this function so that this can be tested easily.